sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryToTableIT.java

```diff
@@ -59,8 +60,9 @@
 
   private BigQueryToTableOptions options;
   private String project;
-  private static final String BIG_QUERY_DATASET_ID =
-      "bq_query_to_table_" + Long.toString(System.currentTimeMillis());
+
+  //Naming format: bg_query_to_table_+ timestamp + 3-digit random number
+  private String bigQueryDatasetId = "bq_query_to_table_";
```

Comment:
Does setting the string here add any use? It is a little cleaner to leave it uninitialized and set the whole format in `@Before`.

Reply:
+1
It is also incorrect name until we get to @before, since it is not a whole ID, but just prefix.